src/middlewared/middlewared/auth.py

```diff
@@ -38,6 +40,7 @@ class UserSessionManagerCredentials(SessionManagerCredentials):
     def __init__(self, user):
         self.user = user
         self.allowlist = Allowlist(user["privilege"]["allowlist"])
+        self.is_user_session = True
```

Comment:
This will fail for UI as they generate a token for currently authenticated user and then use that token (so it'll be `TokenSessionManagerCredentials`.

Let's make this a method instead, say, `use_no_authz_required`. Default implementation will return `False`, `UserSessionManagerCredentials` will return `True`, and `TokenSessionManagerCredentials` will return the value for its parent credentials.

Reply:
Okay. Probably I'll update PR tomorrow.